ops/framework.py

```diff
@@ -945,8 +968,9 @@ def breakpoint(self, name: Optional[str] = None):
             pdb.Pdb().set_trace(code_frame)
         else:
             logger.warning(
-                "Breakpoint %r skipped (not found in the requested breakpoints: %s)",
-                name, indicated_breakpoints)
+                f"Breakpoint {name!r} skipped (not found in the requested "
```

Comment:
Another case of where I think we should still use lazy %-formatting for logging (i.e., let's revert this).